Migration.md

```diff
@@ -181,97 +184,97 @@ release will remove the deprecated code.
 
 1. **Helpers.hh**
     + ***Deprecation:*** IGN_DBL_MAX
-    + ***Replacement:*** ignition::math::MAX_D
+    + ***Replacement:*** gz::math::MAX_D
```

Comment:
nit: I think it's confusing to our history to change these c++ namespaces to refer to `gz` before that was actually in use